src/Evolution/Systems/Cce/WorldtubeDataManager.hpp

```diff
@@ -20,6 +20,20 @@
 
 namespace Cce {
 
+namespace detail {
+
+template <typename InputTags>
+void puper(size_t& time_span_start, size_t& time_span_end,
+           Variables<InputTags>& coefficients_buffers,
+           Variables<InputTags>& interpolated_coefficients, size_t buffer_depth,
+           size_t interpolator_length, size_t l_max);
+
+template <typename InputTags>
+void constructor(size_t& buffer_depth,
```

Comment:
Rename to `initialize_buffers`. Also take the reference arguments as `gsl::not_null`